ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -316,6 +317,33 @@ const GENE_DISEASE_DETAIL_SECTIONS = [
 
 const RNA_SEQ_DETAIL_FIELDS = ['zScore', 'pValue', 'pAdjust']
 
+const INDIVIDUAL_NAME_COLUMN = { name: 'individualName', content: '', format: ({ individualName }) => (<b>{individualName}</b>) }
+
+const RNA_SEQ_COLUMNS = [
+  INDIVIDUAL_NAME_COLUMN,
+  ...RNA_SEQ_DETAIL_FIELDS.map(name => (
+    { name, content: camelcaseToTitlecase(name).replace(' ', '-'), format: row => row[name].toPrecision(3) }
+  )),
+]
+
+const PHENOTYPE_GENE_INFO_COLUMNS = [
+  { ...INDIVIDUAL_NAME_COLUMN, width: 4 },
```

Comment:
the width in the screenshot is not the width you are specifying in this config, its just setting each column to its own minimum width because of the `singleLine` and `fixedWdith` props. It looks good as it is, but you should take the unused `width` props out here

Reply:
Okay. I'll take out all the widths.